requirements-dev.txt

```diff
@@ -1,7 +1,4 @@
-black==22.3.0
 click==8.0.4
-mypy==1.3.0
```

Comment:
Is this right? thus Ruff also takes care of type hints?

Reply:
No, but I am passing everything to the pre-commit hook: see https://results.pre-commit.ci/run/github/534281694/1706217149.DTa7osJdTE-SvRWjeQDG2Q It is still in

https://github.com/juanitorduz/pymc-bart/blob/18554ab35a4dbfce8bd74278fbffa2cb216c8d71/.pre-commit-config.yaml#L21-#L27

with the specific version